### PR TITLE
App Library Remote Control Manager

### PR DESCRIPTION
**Pull request by joeljfischer**
This proposal adds a remote control manager to the app libraries which manages everything related to the remote control systems on head units and allows apps to more conveniently retrieve and set remote control data.

### CODE REVIEW

**Comment by joeljfischer** [comment]
@pelsila This proposal is ready for review

File: proposals/nnnn-app-lib-remote-control-manager.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+public void setUserSeatLocation(Grid location, RemoteControlCompletionListener completionListener)
+```
+
+##### JavaScript Suite
+```js
+async setUserSeatLocation(location)
+```
+
+#### Getting User Consent
+On RPC 6.0+ systems, an app may be required to request access to a remote control module from the user, but it should only do when it actually needs that access. The developer will be able to request user consent manually for a module or modules whenever it makes sense within their app's flow.
```
Comment (pelsila) [inline proposals/nnnn-app-lib-remote-control-manager.md:68]:
```suggestion
On RPC 6.0+ systems, an app may be required to request access to a remote control module from the user, but only when it actually needs that access. The developer will be able to request user consent manually for a module or modules whenever it makes sense within their app's flow.
```

**Review by pelsila — CHANGES_REQUESTED** [review]
@joeljfischer Just one minor suggestion.  Let me know if you have any questions.  Thank you

**Comment by joeljfischer** [comment]
@pelsila Committed the suggestion, thank you for the review.

**Review by pelsila — APPROVED** [review]

**Pull request merged**